python/datasources/cdc_restricted.py

```diff
@@ -27,21 +27,25 @@ def write_to_bq(self, dataset, gcs_bucket, **attrs):
             raise ValueError('filename passed to write_to_bq is not a '
                              'comma-separated list of files')
         files = gcs_files.split(',')
+        print("Files that will be written to BQ:", files)
```

Comment:
was this for dev?

Reply:
yeah, it was, but honestly i think it's good to keep in as logging. let me know if there's a better way to log things